Makefile

```diff
@@ -1,15 +1,15 @@
-export GOPATH     := $(abspath ../..)
-export NEX        := $(abspath ../../bin/nex)
+export GOPATH     := $(abspath ../../../..)
+export NEX        := $(abspath ../../../../bin/nex)
 
 all: $(NEX) test
 
 $(NEX): main.go nex.go
-	go fmt nex
-	go install nex
+	go fmt github.com/blynn/nex
+	go install github.com/blynn/nex
 
 test: $(NEX) $(shell find test -type f)
-	go fmt nex nex/test
-	go test nex nex/test
+	go fmt github.com/blynn/nex github.com/blynn/nex/test
+	go test github.com/blynn/nex github.com/blynn/nex/test
```

Comment:
Uunrelated, but this makefile doesn't work when you pull the repo via a normal `go get github.com/blynn/nex`.  This only works when you promote `nex` directly to `go/src/nex` which is unusual.  We can revert this if needed.